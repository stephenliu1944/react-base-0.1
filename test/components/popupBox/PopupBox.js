import PopupBox from 'components/popupBox/PopupBox';

it('check PopupBox.render()', () => {
    expect(PopupBox.render()).toMatchSnapshot();
});

// TODO: