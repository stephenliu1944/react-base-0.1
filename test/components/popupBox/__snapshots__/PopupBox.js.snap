// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check PopupBox.render() 1`] = `
"
            <div class=\\"Arrow\\"></div>
            <div class=\\"PopupBox-header\\">
                <p class=\\"PopupBox-title\\">Separate multipie resource name with commas</p>
                <i id=\\"closePopupBox\\" class=\\"Icon icon-close\\"></i>
            </div>
            <div class=\\"PopupBox-body\\">
                <input id=\\"popupBoxInput\\" class=\\"PopupBox-Input\\" type=\\"text\\" placeholder=\\"Input value\\" value=\\"\\" />
                <p id=\\"popupBoxError\\" class=\\"PopupBox-error\\"></p>
            </div>
            <div class=\\"PopupBox-footer\\">
                <a id=\\"addResources\\" class=\\"Button Button--primary\\">Add Resources</a>
                <a id=\\"cancelResource\\" class=\\"Button Button--cancel\\">Cancel</a>
            </div>
        "
`;
